sidebars.ts

```diff
@@ -348,33 +326,53 @@ const sidebars: SidebarsConfig = {
                     "integrations/data-imports/azure_upload-deprecated",
                   ],
                 },
-                "integrations/gtm",
-                "integrations/event_filtering",
-                "integrations/event_webhook",
-                "integrations/openai",
               ],
+              
             },
 
             {
               "Data Exports": ["integrations/data-exports/overview"],
             },
             {
-              "Edge (Vercel, Fastly, Cloudflare KV)": [
+              "Serverless & Edge": 
```

Comment:
i think we will end up with a flat list of all of these, just like with sdks, otherwise it gets too burried

Reply:
I will do some research on how others do it. 

I agree over-nesting feels bad, but In some cases I think having fewer findable categories actually helps you find what you're looking for faster if you have less to scan.

Reply:
I agree with this - think we should do more research on how others do it 

There are some situations where having the laundry list is just really hard to scan - and nesting can help with findability just because you have less to scan. But don't have a great intuition yet for that balance. Might be overrotating